src/genegraph/database/dataset.clj

```diff
@@ -0,0 +1,98 @@
+(ns genegraph.database.dataset
+  "Namespace for handling operations on persistent Jena datasets.
+  Specifically designed around handling asychronous writes. "
+  (:require [clojure.spec.alpha :as spec]
```

Comment:
This doesn't appear to be used currently.

Reply:
Right now my plan is to use spec to describe and validate the expected parameters for dataset commands and the parameters for opening a dataset. Have not put that in yet though...